suite-common/wallet-utils/src/balanceUtils.ts

```diff
@@ -25,11 +24,13 @@ export const formatCoinBalance = (value: string) => {
         // indicate the dust
         const noDecimalsLeft = fixedBalanceBig.modulo(2).toFixed() === '0';
         if (noDecimalsLeft) {
-            return fixedBalanceBig.toFixed(2);
+            return formatCurrencyAmount(fixedBalanceBig.toNumber(), locale, 2);
         }
 
-        return isTruncated ? `${fixedBalanceBig.toFixed()}…` : fixedBalanceBig.toFixed();
+        const localizedBalance = formatCurrencyAmount(fixedBalanceBig.toNumber(), locale);
+
+        return isTruncated ? `${localizedBalance}…` : localizedBalance;
```

Comment:
Would be nice to show tooltip with the whole amount in some follow-up issue.

Reply:
I think we should discuss it later again together with @hynek-jina and @sime